src/Ormolu/Printer/Combinators.hs

```diff
@@ -270,3 +277,28 @@ comma = txt ","
 
 space :: R ()
 space = txt " "
+
+----------------------------------------------------------------------------
+-- Layout
+
+data LayoutOptions
+  = LayoutOptions
+      { loSit :: Bool
+      }
+
+defaultLayoutOptions :: LayoutOptions
```

Comment:
I'm not sure if you win anything with the `LayoutOptions` here. It adds verbosity, but maybe it would be better to have a sum type isomorphic to `Bool` here just with more meaningful names for data constructors? Do you plan to extend `LayoutOptions` with other parameters?

Reply:
You are right. Initially, that `LayoutOptions` had two more fields, but I managed to remove them on later iterationsm. I'll make it a sum type type.